proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -161,6 +164,7 @@ message Event {
     PayloadRepositoryLabelCreated repository_label_created = 35;
     PayloadRepositoryLabelMoved repository_label_moved = 36;
     PayloadCuratedPluginDeleted curated_plugin_deleted = 37;
+    PayloadServerBreakingChangesChanged server_breaking_changes_changed = 38;
```

Comment:
Don't like the `_changed` stutter, but it keeps it consistent with the rest of the definitions.

The alternative is to refer to this as `server_breaking_changed`, but don't think it's worth changing.

Reply:
What about "Server Breaking Protection Policy", so it'd result in `PayloadServerBreakingProtectionPolicyChanged`

Reply:
```
PayloadServerBreakingChangePolicyEnabled
PayloadServerBreakingChangePolicyDisabled
```